README.md

```diff
@@ -15,54 +15,59 @@ Now:
 * add expRESTive to your _package.json_ file: `$ npm install --save exprestive`
 * add the expRESTive middleware to your application
 
-    ```coffeescript
-    express = require 'express'
-    exprestive = require 'exprestive'
+    ```javascript
+    express = require('express')
+    exprestive = require('exprestive')
```

Comment:
should these be prefixed with `const`? Same for `app` a couple line below and the small example in the options section